src/jgikbase/idmapping/storage/mongo/id_mapping_mongo_storage.py

```diff
@@ -76,7 +76,16 @@
                                      (_FLD_PRIMARY_ID, 1),
                                      (_FLD_SECONDARY_NS, 1),
                                      (_FLD_SECONDARY_ID, 1)],
-                             'kw': {'unique': True}}],
+                             'kw': {'unique': True}
+                             },
+                            # index for 'backwards' queries
+                            # could improve performance by including the primary IDs for covered
```

Comment:
Can you measure the size of indexes that get created?

Reply:
Yeah. I'm just avoiding premature optimization here. It's easy to add an index if it turns out the 'reverse' queries are much slower than the forward queries.